packages/types/src/ui5App.ts

```diff
@@ -22,6 +22,11 @@ export interface App {
 	localStartFile?: string;
 }
 
+export enum BootstrapSrc {
+	CdnOpenUI5 = "Content delivery network (OpenUI5)",
+	CdnSAPUI5 = "Content delivery network (SAPUI5)",
+	Local = "Local Resources"
+}
```

Comment:
I am not in favor of this approach. This looks like a question for the prompting and based on that the template should be called with the location/url of the UI5 sources. Such an approach would also allow to point to a specific SAP system for testing with its UI5 version.

Would this work for `easyUI5`?

Reply:
actually the enum is aligned with exactly such a prompt in easy-ui5:
https://github.com/ui5-community/generator-ui5-project/blob/b7ebdfcefd0dd022a9c1eacf796dd3646bbf9368/generators/app/index.js#L64

Reply:
I meant, would it also work for `easyUI5` to keep their prompts and then convert it into the target url for UI5? The template could then use the provided url in the proxy middleware.